py/selenium/webdriver/remote/errorhandler.py

```diff
@@ -110,7 +110,7 @@ def check_response(self, response):
                             value = message
                             try:
                                 message = message['message']
-                            except TypeError:
+                            except KeyError:
```

Comment:
could you change this to instead of being a `try.. except` be

`message = message.get('message')`